.github/workflows/benchmark_cpu_weekly.yml

```diff
@@ -190,7 +190,7 @@ jobs:
     with:
       command: boolean
       additional_recipe: measure_boolean_key_sizes
-      additional_file_to_parse: boolean_key_size.csv
+      additional_file_to_parse: boolean_key_sizes.csv
```

Comment:
file name has been validated ?

Reply:
Here's the name in the codebase:
`let results_file = Path::new("boolean_key_sizes.csv");`